R/kiss-rule.R

```diff
@@ -110,7 +110,7 @@ asJson.KissRule.Event <- function(rule) {
     "type":"{{type}}",
     "negate": {{negate}},
     "event":{{event}},
-    "frequencyValue":{{frequencyValue}},
+    "frequencyValue":"{{frequencyValue}}",
```

Comment:
Is this a mistake? I think Yosem had fixed this by removing quotes. Need to check.